query/mysql/executor.go

```diff
@@ -268,15 +274,21 @@ func (e *QueryExecutor) showCreate(dbTable string) (string, error) {
 	var tableName string
 	var tableDef string
 	err := e.conn.DB().QueryRow("SHOW CREATE TABLE "+dbTable).Scan(&tableName, &tableDef)
+	if err == sql.ErrNoRows {
+		err = fmt.Errorf("table %s doesn't exist ", dbTable)
+	}
 	return tableDef, err
 }
 
 func (e *QueryExecutor) showIndex(dbTable string) (map[string][]proto.ShowIndexRow, error) {
 	rows, err := e.conn.DB().Query("SHOW INDEX FROM " + dbTable)
-	if err != nil {
+	if err != nil && err != sql.ErrNoRows {
 		return nil, err
 	}
 	defer rows.Close()
+	if err == sql.ErrNoRows {
+		err = fmt.Errorf("table %s doesn't exist", dbTable)
+	}
```

Comment:
Shouldn't we just return this `err` here?
Looks like it will be overwritten below.